### PR TITLE
Cooldown – Prevent `moment` bundling

### PR DESCRIPTION
**Pull request by ismaeldcom**
#### Changes proposed in this Pull Request
We rely on WP/WC `dependency-extraction-webpack-plugin` to reuse existing libraries and avoid bundling them. But due to a malformed import, we are including `moment` in our bundle. As it was imported as `moment/moment`, the plugin wasn't able to externalize it.

#### Testing instructions
Code change makes sense. There is no functional difference.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `5949` or branch name `dev/prevent-moment-bundling` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=dev/prevent-moment-bundling) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: bd9261871f905f28a613fbf26c04796503663731
- Build time: 2023-04-03 13:04:42 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/overview/modal/update-business-details/index.tsx [hunk 1]
```diff
@@ -12,7 +13,6 @@ import strings from './strings';
 import { useState } from '@wordpress/element';
```
Comment (oaratovskyi) [inline client/overview/modal/update-business-details/index.tsx]: nitpick: we could also move lines 13-15 to External dependencies section

**Review by oaratovskyi — APPROVED** [review]
Thanks @ismaeldcom, looks good to me!

File: client/overview/modal/update-business-details/index.tsx (hunk 1 above)
Comment (ismaeldcom) [reply client/overview/modal/update-business-details/index.tsx]: Good catch! In fact `useState` should be imported from `React` in this case. I've also updated every other import section as part of bd92618

**Pull request merged**